rowboat/plugins/infractions.py

```diff
@@ -909,24 +1107,36 @@ def kick(self, event, user, reason=None):
                 u=member.user,
             ))
         else:
-            raise CommandFail('invalid user')
+            raise CommandFail('Invalid user')
 
     @Plugin.command('mkick', parser=True, level=CommandLevels.MOD)
-    @Plugin.parser.add_argument('users', type=long, nargs='+')
+    @Plugin.parser.add_argument('users', type=str, nargs='+')
```

Comment:
this one had worked before, any particular reason for the change?

Reply:
Just inline with all the other mass ones and we can handle the types etc in the function itself

Reply:
👍